src/sparseml/transformers/export.py

```diff
@@ -286,19 +286,37 @@ def _parse_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def main():
-    args = _parse_args()
-    _LOGGER.info(f"Exporting {args.model_path} to ONNX")
+def main(
+    task: str,
+    model_path: str,
+    sequence_length: int,
+    no_convert_qat: bool,
+    finetuning_task: str,
+    onnx_file_name: str,
+):
+    _LOGGER.info(f"Exporting {model_path} to ONNX")
```

Comment:
consider removing this to avoid over logging

Reply:
Also fixing the failing integration tests :)

Reply:
Thanks @kylesayrs! You da man.
<img src="https://media4.giphy.com/media/AMQgg6x2QLCFoG1Whr/giphy.gif"/>